src/components/Asset/AssetActions/Download.tsx

```diff
@@ -73,7 +73,11 @@ export default function Download({
     setValidOrderTx(asset?.accessDetails?.validOrderTx)
     // get full price and fees
     async function init() {
-      if (asset?.accessDetails?.addressOrId === ZERO_ADDRESS) return
+      if (
+        asset?.accessDetails?.addressOrId === ZERO_ADDRESS ||
```

Comment:
I couldn't verify the need for this. In which case does the address equal to ZERO_ADDRESS? is this really need it?

Reply:
I don't know why that check is in there, that's why I didn't touch it.

Reply:
that check is for the publish preview